great_expectations/core/http_handler.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+import requests
+
+from great_expectations import __version__
+
+
+class HTTPHandler:
+    def __init__(self, access_token: str, timeout: int = 20) -> None:
+        self._headers = self._init_headers(access_token)
+        self._timeout = timeout
+
+    def _init_headers(self, access_token: str) -> dict:
+        headers = {
+            "Content-Type": "application/vnd.api+json",
+            "Authorization": f"Bearer {access_token}",
+            "Gx-Version": __version__,
+        }
+        return headers
+
+    def get(self, url: str, params: Optional[dict] = None) -> requests.Response:
```

Comment:
How do we abstract away the very specific `requests.Response` return? Perhaps a wrapper `HTTPResponse` class that interacts with the underlying `requests.Response` type?

Reply:
I think that is a great idea.